doc/21-development.md

```diff
@@ -304,6 +307,18 @@ Adjust the coredump kernel format and file location on Linux:
 
     sysctl -w kernel.core_pattern=/var/lib/cores/core.%e.%p
 
+Alternatively, to make the changes persistent across reboots, add them
+to `/etc/sysctl.conf`. You can use the `sysctl` command to apply the changes
+without reboot. You have to create the directory where the core dumps will
+be written to yourself.
+
+    vim /etc/sysctl.conf
+
+    fs.suid_dumpable = 1
+    kernel.core_pattern=/var/lib/cores/core.%e.%p
```

Comment:
systemd-coredump is installing its own config to `/usr/lib/sysctl.d/50-coredump.conf`.

This should only be overwritten on systems where `systemd-coredump` is not installed.